libraries/AP_MSP/AP_MSP_Telem_Backend.cpp

```diff
@@ -577,6 +577,19 @@ void AP_MSP_Telem_Backend::msp_handle_airspeed(const MSP::msp_airspeed_data_mess
 #endif
 }
 
+uint32_t AP_MSP_Telem_Backend::get_osd_flight_mode_bitmask(void)
+{
+    uint32_t mode_mask = 0;
+    const AP_Notify& notify = AP::notify();
```

Comment:
Any reason why we are using AP_Notify instead of using the more regular, "hal.util->get_soft_armed()"?  I would prefer we don't use the AP_Notify class as a method to shift data from one part of the system to another.

Reply:
Before moving some code to the base class I also needed failsafe status so I used AP_Notify to get both, not needed anymore